code/script.js

```diff
@@ -1,53 +1,273 @@
-// DOM selectors (variables that point to selected DOM elements) goes here 👇
-const chat = document.getElementById('chat')
+// DOM selectors 👇
+const chat = document.getElementById("chat")
+const submitButton = document.getElementById("send-btn")
+const nameInput = document.getElementById("name-input")
+const form = document.getElementById("form")
+const inputWrapper = document.getElementById("input-wrapper")
```

Comment:
This is not used. Seems like you're using the form instead, which is fine - but please remove any unused code